src/tests/Services/update-user.service.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { TestBed, inject } from '@angular/core/testing';
```

Comment:
Can it have a test case to check if the UI is updated?

Reply:
Done.